queued_search/management/commands/process_search_queue.py

```diff
@@ -284,8 +284,10 @@ def handle_deletes(self):
             pks = []
 
             for obj_identifier in obj_identifiers:
-                current_index.remove_object(obj_identifier, using=self.using)
-                pks.append(self.split_obj_identifier(obj_identifier)[1])
+                pk = self.split_obj_identifier(obj_identifier)[1]
+                instance = self.get_instance(model_class, pk)
```

Comment:
I don't think you want this change. It will cause a db query for deleted objects which will raise an error.